src/layouts/default.hbs

```diff
@@ -61,52 +61,52 @@
                 </div>
 
                 <nav class="infusion-docs-fluidCategories small-12 medium-9 column">
                     {{! Highlight the (active) category the document belongs to. }}
                     {{#each siteStructure}}
-                        {{#ifEqual category ../document.category}}
-                            <a href="{{{relativeUrl categoryHref ../../document.url}}}" title="{{category}}" class="infusion-docs-category-active">{{category}}</a>
+                        {{#ifEqual category ../category}}
+                            <a href="{{ url categoryHref }}" title="{{category}}" class="infusion-docs-category-active">{{category}}</a>
                         {{else}}
-                            <a href="{{{relativeUrl categoryHref ../../document.url}}}" title="{{category}}">{{category}}</a>
+                            <a href="{{ url categoryHref }}" title="{{category}}">{{category}}</a>
                         {{/ifEqual}}
                     {{/each}}
                 </nav>
             </header>
 
             <div class="infusion-docs-mainBody">
                 <nav id="tableOfContents" class="infusion-docs-TOC small-12 medium-3 column" tabindex="-1">
-                    {{#ifEqual document.category "Search"}}
+                    {{#ifEqual category "Search"}}
                     {{else}}
                         <span class="infusion-docs-category">Search</span>
                         <p class="docs-search-mini"><input type="text" name="qs"/></p>
 
                         {{#each siteStructure}}
                         {{! Output only the table of contents matching the current document's category }}
-                            {{#ifEqual category ../document.category}}
+                            {{#ifEqual category ../category}}
                                 <span class="infusion-docs-category">{{category}}</span>
                                 {{#each sections}}
                                     <span class="infusion-docs-sectionName">{{sectionName}}</span>
                                     <ul>
                                         {{#each pages}}
                                             <li>
-                                                {{#ifEqual href ../../../../document.url}}
+                                                {{#ifEqual href ../../../page.url}}
                                                     <span class="infusion-docs-TOC-pageName">{{pageName}}</span>
                                                 {{else}}
-                                                    <a href="{{relativeUrl href ../../../../../document.url}}"><span class="infusion-docs-TOC-pageName">{{pageName}}</span></a>
+                                                    <a href="{{ url href }}"><span class="infusion-docs-TOC-pageName">{{pageName}}</span></a>
                                                 {{/ifEqual}}
```

Comment:
This appears to be broken. For reference, compare:

Old version: https://docs.fluidproject.org/infusion/development/understandinginfusioncomponents
New version: https://deploy-preview-187--fluid-infusion-docs.netlify.app/infusion/development/understandinginfusioncomponents

Note that in the old version, the "Understanding Infusion Components" link in the left sidebar navigation is turned into a span and highlighted to indicate the current page. There's no differentiation from the other sidebar links in the new version.

Reply:
This appears to have crawled in some commit previously because I specifically remember taking care of this. Anyway, thanks for noticing, I have fixed them now. Please take a look.